code/game/gamemodes/game_mode.dm

```diff
@@ -374,7 +373,6 @@
 
 /datum/game_mode/New()
 	newscaster_announcements = pick(newscaster_standard_feeds)
-	cultdat = setupcult()

```

Comment:
Why. Why did this run every single time a new game-mode was created?

Reply:
at the time if it didn't..everything broke...cuase of how things loaded..like that cult ruin on the asteroid and loading icons.